setup.py

```diff
@@ -1,23 +1,24 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-    Setup file for core.
+    Setup file for f0cal.
     Use setup.cfg to configure your project.
 
-    This file was generated with PyScaffold 3.2.3.
+    This file was generated with PyScaffold 3.1.
```

Comment:
@AndreyShprengel Older version of PyScaffold? Something weird going on here.

Reply:
This makes sense as you said you started a new pyscaffold project, likely using a newer version of pyscaffold, the core2 branch here has an exact replica of the original (working) core/f0cal package. I'm not sure how much the pyscaffold version matters. My thought is that we should first switch over to this new repo with working code, and then do any refactors after. However if you want to do a refactor in one step we can do that here as well.

Reply:
There are a bunch of other things in the `setup.cfg` diff that need resolving. Probably better to import just the source code and a leave the PyScaffold boilerplate alone.